Framework/CurveFitting/test/FuncMinimizers/FABADAMinimizerTest.h

```diff
@@ -190,23 +190,23 @@ class FABADAMinimizerTest : public CxxTest::TestSuite {
     TS_ASSERT_EQUALS(PDF->getNumberHistograms(), nParams + 1);
     TS_ASSERT_EQUALS(PDF->x(0).size(), 21);
     TS_ASSERT_EQUALS(PDF->y(0).size(), 20);
-    TS_ASSERT_DELTA(PDF->y(0)[7], 0.41, 0.01);
-    TS_ASSERT_DELTA(PDF->y(1)[8], 5.24, 0.01);
-    TS_ASSERT_DELTA(PDF->y(2)[0], 0.44, 0.01);
+    TS_ASSERT_DELTA(PDF->y(0)[7], 0.41, 0.3);
+    TS_ASSERT_DELTA(PDF->y(1)[8], 5.24, 1.0);
+    TS_ASSERT_DELTA(PDF->y(2)[0], 0.44, 0.3);
```

Comment:
That makes sense. However, I think it may be worth adding a long running test to get the deterministic result. See https://github.com/mantidproject/mantid/pull/19973/files#diff-2d30ab47a9ebbc4ecf8de180cf7d334aR41 everything after the exclude function is not used in PRs as they take too long to run